dev, v2.1, v3.0/how-to/deploy: update TiDB Binlog document (#1458)
pingcap/docs: dev/how-to/deploy/tidb-binlog.md

        # the address of the PD cluster nodes
        pd-urls = "http://192.168.0.16:2379,http://192.168.0.15:2379,http://192.168.0.14:2379"

        # [security]
        # This section is generally annotated if no special security settings are required.

>         # The file path containing a list of trusted SSL CAs connected to the cluster

Reviewer: ```suggestion
        # The file path containing a list of trusted SSL CAs connected to the cluster.
```

        # ssl-ca = "/path/to/ca.pem"

>         # The path to the X509 certificate in PEM form that is connected to the cluster

Reviewer: ```suggestion
        # The path to the X509 certificate that is connected to the cluster in PEM format.
```

        # ssl-cert = "/path/to/drainer.pem"

>         # The path to the X509 key in PEM form that is connected to the cluster

Reviewer: ```suggestion
        # The path to the X509 key that is connected to the cluster in PEM format.
```

        # ssl-key = "/path/to/drainer-key.pem"

        # [storage]
        # Set to true (by default) to guarantee reliability by ensuring binlog data is flushed to the disk
        # sync-log = true

        # When the available disk capacity is less than the set value, Pump stops writing data.
        # 42 MB -> 42000000, 42 mib -> 44040192
        # default: 10 gib
        # stop-write-at-available-space = "10 gib"

>         # The LSM DB settings embedded in Pump. Unless you know this part well, it is usually annotated.

Reviewer: **Note:** "注释掉" means "comment out" so that it won't take effect. "annotate" cannot convey this meaning.

```suggestion
        # The LSM DB settings embedded in Pump. Unless you know this part well, it is usually commented out.
```

            the address through which Drainer provides the service (-addr="192.168.0.13:8249")
        -c int
            the number of the concurrency of the downstream for replication. The bigger the value, the better throughput performance of the concurrency (1 by default).
        -cache-binlog-count int

>             the limit on the number of binlogs in the cache (65536 by default)

Reviewer: ```suggestion
            the limit on the number of binlog files in the cache (65536 by default)
```

Reviewer 2: I think the description 'files' is not suitable. In this part 'binlogs' are struct instances but not files. Maybe the description 'items' is better.

Reviewer: @lichunzhu Got it. Thanks for the explanation! 
@TomShawn Then let's use "items" here.

Author: OK.

            It it is not set, the monitoring metrics are not reported.
        -metrics-interval int
            the report frequency of the monitoring metrics (15 by default, in seconds)
        -node-id string

>             the unique ID of a Pump node. If you do not specify this ID, the system automatically generates an ID based on the host name and listening port.

Reviewer: ```suggestion
            the unique ID of a Drainer node. If you do not specify this ID, the system automatically generates an ID based on the host name and listening port.
```

        -pd-urls string
            the address of the PD cluster nodes (-pd-urls="http://192.168.0.16:2379,http://192.168.0.15:2379,http://192.168.0.14:2379")
        -safe-mode

>             Whether to enable safe mode so that the downstream MySQL/TiDB can be repeatedly written

Reviewer: ```suggestion
            Whether to enable safe mode so that data can be written into the downstream MySQL/TiDB repeatedly
```

>             Replace the `INSERT` statement with the `REPLACE` statement and split the `UPDATE` statement into `DELETE` plus `REPLACE`.

Reviewer: ```suggestion
            This mode replaces the `INSERT` statement with the `REPLACE` statement and splits the `UPDATE` statement into `DELETE` plus `REPLACE`.
```

        # If the downstream service is MySQL, set it to "False".
        disable-dispatch = false

>         # safe mode enables the downstream MySQL/TiDB to be written repeatedly.

Reviewer: ```suggestion
        # In safe mode, data can be written into the downstream MySQL/TiDB repeatedly.

>         # replace the `INSERT` statement with the `REPLACE` statement and replace the `UPDATE` statement with `DELETE` plus `REPLACE` statements.

Reviewer: ```suggestion
        # This mode replaces the `INSERT` statement with the `REPLACE` statement and replaces the `UPDATE` statement with `DELETE` plus `REPLACE` statements.
```

        safe-mode = false

        # the downstream service type of Drainer ("mysql" by default)
        # Valid value: "mysql", "kafka", "file", "flash"
        db-type = "mysql"

>         # the `commit ts` of the transaction. If it is in the list, the transaction is filtered and not replicated to the downstream.

Reviewer: ```suggestion
        # If `commit ts` of the transaction is in the list, the transaction is filtered and not replicated to the downstream.
```

        # the Kafka configuration when `db-type` is set to "kafka"
        # [syncer.to]

>         # only one of kafka-addrs and zookeeper-addrs is needed. If both are present, the program gives priority.

Reviewer: ```suggestion
        # only one of kafka-addrs and zookeeper-addrs is needed. If both are present, the program gives priority
```
